ckb/src/main/java/org/nervos/ckb/utils/address/AddressGenerator.java

```diff
@@ -20,7 +21,8 @@ public static String generate(Network network, Script script) {
     String codeHash = Numeric.cleanHexPrefix(script.codeHash);
     String args = Numeric.cleanHexPrefix(script.args);
     if (Script.TYPE.equals(script.hashType)
-        && args.length() == 40
+        && args.length() >= 40
+        && args.length() <= 42
```

Comment:
args max length is 22 bytes, so it should be 44 here?

Reply:
Fixed. https://github.com/nervosnetwork/ckb-sdk-java/pull/374/commits/334b0f18074687e14d2315464a697fba43bfff21